edsby.py

```diff
@@ -443,15 +525,24 @@ def getAllClassAverages(self):
         Returns an object containing all assignment metadata for a specified course, ordered by RID
         This includes the NID, RID, and weighting (points possible) of the assignments, but not your score.
     """
+
     def getClassAssignmentMetadata(self, classNID):
-        return requests.get('https://'+self.edsbyHost+'/core/node.json/'+str(classNID)+'?xds=MyWork&student='+str(self.studentData['unid']),cookies=self.getCookies(),headers=self.getHeaders()).json()['slices'][0]['data']['loaddata']['gradebook']['terms']
+        return requests.get(
+            'https://' + self.edsbyHost + '/core/node.json/' + str(classNID) + '?xds=MyWork&student=' + str(
+                self.studentData['unid']), cookies=self.getCookies(), headers=self.getHeaders()).json()['slices'][0][
+            'data']['loaddata']['gradebook']['terms']
 
     """
         Returns an object containing all assignment scores for a specified course, ordered by NID
         This includes the NID and points earned on the assignment, but nothing else.
     """
+
     def getClassAssignmentScores(self, classNID, classRID):
-        return requests.get('https://'+self.edsbyHost+'/core/node.json/'+str(classNID)+'/'+str(classRID)+'/'+str(classNID)+'?xds=MyWorkAssessmentPane&unit=all&student='+str(self.studentData['unid'])+'&model=24605449',cookies=self.getCookies(),headers=self.getHeaders()).json()["slices"][0]["data"]['grades']
+        return requests.get(
+            'https://' + self.edsbyHost + '/core/node.json/' + str(classNID) + '/' + str(classRID) + '/' + str(
+                classNID) + '?xds=MyWorkAssessmentPane&unit=all&student=' + str(
+                self.studentData['unid']) + '&model=24605449', cookies=self.getCookies(),
```

Comment:
Overall, I do like the improved readability that comes from PyCharm's formatting, but the new formatting for calls to requests is very troublesome to me:

```python
        return requests.get(
             'https://'  self.edsbyHost  '/core/node.json/'  str(classNID)  '/'  str(classRID)  '/'  str(
                 classNID)  '?xds=MyWorkAssessmentPane&unit=all&student='  str(
                 self.studentData['unid'])  '&model=24605449', cookies=self.getCookies(),
             headers=self.getHeaders()).json()["slices"][0]["data"]['grades']
```

- The URL is all over the place, weirdly broken up over multiple lines,
- `str()` casts are broken off into multiple lines, 
-  Arguments being passed to requests are mixed in with the URL and JSON parser.

So that format isn't the easiest to follow, in my opinion.

Would it be possible for PyCharm to generate something similar to the following?

```python
        return requests.get(
            'https://'+ self.edsbyHost +'/core/node.json/'+ str(classNID) +'/'+ str(classRID) +'/'+ str(classNID) +'?xds=MyWorkAssessmentPane&unit=all&student='+ str(self.studentData['unid']) +'&model=24605449',
             cookies=self.getCookies(),headers=self.getHeaders()
        ).json()["slices"][0]["data"]['grades']
```

I'd much prefer this format, because I find it easier to follow for a few reasons:
- the full URL to the API endpoint is on its own line, which is much easier to read/understand. The spacing between the variable names and strings we're concatenating them with makes them stand out more, so each component is easy to follow along with,
-  kwargs being passed to `requests.get` have their own line, which makes it easier to follow along with. From a development standpoint, as well, since centralizing that makes it easier to add/modify those args,
- json parsing, as well as accessing the property paths, is also now on its own line.